vector/src/main/java/im/vector/activity/VectorHomeActivity.java

```diff
@@ -473,11 +491,14 @@ public void run() {
      * Display the TAB if it is required
      */
     private void showFloatingActionButton() {
-        if (null != mFloatingActionButton) {
+        if (null != mFloatingActionsMenu) {
             if ((mCurrentMenuId == R.id.bottom_action_favourites) || (mCurrentMenuId == R.id.bottom_action_groups)) {
-                mFloatingActionButton.setVisibility(View.GONE);
+                mFloatingActionsMenu.collapse();
+                mFloatingActionsMenu.setVisibility(View.GONE);
             } else {
-                mFloatingActionButton.show();
+                mFloatingActionsMenu.collapse();
+                mFloatingActionsMenu.setVisibility(View.VISIBLE);
```

Comment:
`show()` display the button with an animation. Is the animation still there with `setVisibility()` on a `FloatingActionsMenu`?

Reply:
Implemented animations.